contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -1197,208 +1198,196 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
+          [uint256(0), 10, 0, 0, 0, 90, 0, 0] // stake on each tranche, as % out of the deposit
         ),
-        params
+        productInitParams
       );
-    } else if (stakerAddress == ARMOR) {
-      stakerTrancheRatios = [uint256(20), 25, 25, 15, 10, 0, 0, 0];
+    } else if (stakerAddress == ARMOR_STAKER) {
 
-      uint aaaLowRiskPoolDeposit = 75 * deposit / 100;
-      uint maxFee = 25;
-      uint initialFee = 15;
+      uint armorAAALowRiskPoolDeposit = 75 * deposit / 100;
       migrateToPool(
         StakingPoolMigrationData(
-          initialFee,
-          maxFee,
-          stakerTrancheRatios,
-          aaaLowRiskPoolDeposit,
-          ARMOR,
+          ARMOR_STAKER, // stakerAddress
+          ARMOR_MANAGER, // managerAddress
+          '', // ipfsDescriptionHash --- TODO fill in
           false, // isPrivatePool
-          '', // ipfsDescriptionHash
-          ARMOR_MANAGER // managerAddress
+          15, // initialPoolFee
+          25, // maxPoolFee
+          armorAAALowRiskPoolDeposit, // deposit
+          [uint256(20), 25, 25, 15, 10, 0, 0, 0] // stake on each tranche, as % out of the deposit
         ),
-        params
+        productInitParams
       );
-      uint aaRiskPoolDeposit = deposit - aaaLowRiskPoolDeposit;
 
+      uint armorAAMidRiskPoolDeposit = deposit - armorAAALowRiskPoolDeposit;
       migrateToPool(
         StakingPoolMigrationData(
-          initialFee,
-          maxFee,
-          stakerTrancheRatios,
-          aaRiskPoolDeposit,
-          ARMOR,
+          ARMOR_STAKER, // stakerAddress
+          ARMOR_MANAGER, // managerAddress
+          '', // ipfsDescriptionHash --- TODO fill in
           false, // isPrivatePool
-          '', // ipfsDescriptionHash
-          ARMOR_MANAGER // managerAddress
+          15, // initialPoolFee
+          25, // maxPoolFee
+          armorAAMidRiskPoolDeposit, // deposit
+          [uint256(20), 25, 25, 15, 10, 0, 0, 0] // stake on each tranche, as % out of the deposit
         ),
-        params
+        productInitParams
       );
+    } else if (stakerAddress == NM_FOUNDATION) {
 
-    } else if (stakerAddress == NEXUS_FOUNDATION) {
-
-      stakerTrancheRatios = [uint256(0), 0, 0, 0, 0, 0, 0, 0];
       // TODO: when switching the StakingPool manager is supported, simply make LegacyPooledStaking the manager
       // make the deposits and then switch the manager to the foundation
-      // stakerTrancheRatios = [uint256(0), 25, 0, 25, 0, 50, 0, 0];
-
-      // TODO: waiting for final value for maxPoolFee
+      // trancheStakeRatio = [uint256(0), 25, 0, 25, 0, 50, 0, 0];
       migrateToPool(
         StakingPoolMigrationData(
+          NM_FOUNDATION, // stakerAddress
+          NM_FOUNDATION, // managerAddress
+          '', // ipfsDescriptionHash --- TODO fill in
+          true, // isPrivatePool
           0, // initialPoolFee
-          20, // maxPoolFee
-          stakerTrancheRatios,
+          99, // maxPoolFee
```

Comment:
Is this 99 still a dummy value for now?

Reply:
nope. it should be the max we can set this value to. we can't set it to 100, as we end up with division by 0 issues

Reply:
cool, will need to change the expected in the migration-v2.js then, it's set at 20